packages/gestalt/src/Video.js

```diff
@@ -377,7 +374,7 @@ export default class Video extends React.PureComponent<Props, State> {
   };
 
   // Sent when the video is switched to/out-of fullscreen mode
-  handleFullscreenChange = (event: SyntheticEvent<HTMLDivElement>) => {
+  handleFullscreenChange = (event: Event) => {
```

Comment:
This is changing because `document.addEventListener` actually shuffles the native `Event` along instead of the React `SyntheticEvent` so this is the correct typing based on the updates above.